addons/website/models/mixins.py

```diff
@@ -272,7 +480,7 @@ def _search_website_published(self, operator, value):
         is_published = [('is_published', '=', value)]
         if current_website_id:
             on_current_website = self.env['website'].website_domain(current_website_id)
-            return (['!'] if value is False else []) + expression.AND([is_published, on_current_website])
```

Comment:
@robinlej That was a bug. We've seen this problem with @bso-odoo. I'll probably create a task to dig further, sometimes a "search_details" make a raw SQL call with computed fields, it should crash since the computed field doesn't exist in the database... I'll keep you in touch.